src/plugins/customScreenShare.desktop/index.tsx

```diff
@@ -0,0 +1,198 @@
+        restartNeeded: false,
+    },
+    targetBitrate: {
+        description: "Bitrate",
+        default: 600000,
+        type: OptionType.NUMBER,
+        hidden: true
+    },
+    minBitrate: {
+        description: "Bitrate",
+        default: 500000,
+        type: OptionType.NUMBER,
+        hidden: true
+    },
+    maxBitrate: {
+        description: "Bitrate",
+        default: 8000000,
+        type: OptionType.NUMBER,
+        hidden: true
+    },
+});
+
+export default definePlugin({
+    name: "CustomScreenShare",
+    description: "Stream any resolution and any FPS!",
+    authors: [Devs.KawaiianPizza],
+    settingsAboutComponent: () => (
+        <Forms.FormSection>
+            <Forms.FormTitle tag="h3">Usage</Forms.FormTitle>
+            <Forms.FormText>
+                Adds a slider for the quality and fps options submenu
+            </Forms.FormText>
+        </Forms.FormSection>),
+    settings,
+    patches: [
+        {
+            find: "ApplicationStreamSettingRequirements)",
+            replacement: {
+                match: /for\(let . of ..ApplicationStreamSettingRequirements\).+?!1/,
+                replace: "return !0"
+            }
+        },
+        {
+            find: "ApplicationStreamFPSButtonsWithSuffixLabel.map",
+            replacement: {
+                match: /(.=)(.{19}FPS.+?(\i).{11}>(\i).\i,(\i),\i,([A-z.]+).+?\}\)),/,
+                replace: "$1[$self.CustomRange($4,$5,$3,$6,'fps'),...$2],"
+            }
+        },
+        {
+            find: "ApplicationStreamResolutionButtonsWithSuffixLabel.map",
+            replacement: {
+                match: /(.=)(.{19}Resolution.+?(\i).{11}>(\i).\i,\i,(\i),([A-z.]+).+?\}\));/,
+                replace: "$1[$self.CustomRange($4,$3,$5,$6,'resolution'),...$2];"
+            }
+        },
+        {
+            find: "\"remoteSinkWantsPixelCount\",\"remoteSinkWantsMaxFramerate\"",
+            replacement: {
+                match: /(max:|\i=)4e6,/,
+                replace: "$18e6,"
+            }
+        },
+        {
+            find: "\"Discord_Clip_\".concat",
+            replacement: {
+                match: /(=15e)3/, // disable discord idle fps reduction
+                replace: "$18"
+            }
+        },
+        {
+            find: "updateRemoteWantsFramerate(){",
+            replacement: {
+                match: /updateRemoteWantsFramerate..\{/, // disable discord mute fps reduction
+                replace: "$&return;"
+            }
+        },
+        {
+            find: "{getQuality(",
+            replacement: {
+                match: /(bitrateMin:).+?(,bitrateMax:).+?(,bitrateTarget:).+?,/,
+                replace: "$1$self.getMinBitrate()$2$self.getMaxBitrate()$3$self.getTargetBitrate(),"
+            }
+        },
+        {
+            find: "ApplicationStreamResolutionButtonsWithSuffixLabel.map",
+            replacement: {
+                match: /(stream-settings-resolution-.+?children:.)/,
```

Comment:
Does this even produce valid JS if `children` is a variable and not an array? If it is an array:
```suggestion
                match: /(stream-settings-resolution-.+?children:\[)/,
```